src/app/content/highlights/components/cardUtils.ts

```diff
@@ -71,3 +73,121 @@ export const generateUpdatePayload = (
 
   return {updatePayload, preUpdateData};
 };
+
+/**
+ * Calculate positions for the cards according to the location of associated highlights in the document.
+ * Positions will be adjusted to the presence of the other cards in cases like multiple highlights in the same line.
+ */
+const updateStackedCardsPostions = (
+  highlightsElements: Highlight[],
+  heights: Map<string, number>,
+  getHighlightPosition: (highlight: Highlight) => { top: number, bottom: number },
+  initialPositions?: Map<string, number>,
+  addMarginBottomToTheFirstCard = false,
```

Comment:
What determines this? Just want to make sure I understand.

Reply:
I've changed it to `addAditionalMarginForTheFirstCard`

When we calculate `stackedTopOffset` on line `97` we want to make sure that it includes minimum margin-bottom from the previous card. 
I'm adding this margin to all cards except the first one because there is nothing above it.
This logic has changed since now I'm using this function two times - the second call is only for the cards after the `focusedHighlight` so I actually want the first card to be adjusted by this margin-bottom value